app/components/ScanningButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component } from 'react';
+
+import * as React from 'react';
+import { Pressable, Text } from 'react-native';
+import styles from './styles/ScanningButton.style';
+
+import * as Speech from 'expo-speech';
+
+class ScanningButton extends Component {
+  render() {
+    return (
+      <Pressable
+        style={styles.buttonContainer}
+        android_ripple={{ color: '#fff' }}
```

Comment:
Can the ripple colour also be added to the style sheet?

Reply:
You can't move it to the style sheet, but I have moved it to the `DefaultColors.tsx` file so we can modify it easily in the future if we want to